integration-tests/bats/dump.bats

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+}
+
+teardown() {
+    assert_feature_version
+    teardown_common
+}
+
+@test "dump: dolt dump SQL export" {
```

Comment:
I think it's wise to test with a database that uses a few more SQL features. Some things to add

1. Multiple tables
2. Indexes
3. Foreign Keys
4. Views
5. Keyless Tables